source/renderer/app/api/ada/lib/v1/request.js

```diff
@@ -79,13 +79,27 @@ function typedRequest<Response>(
           // When deleting a wallet, the API does not return any data in body
           // even if it was successful
           const { statusCode, statusMessage } = response;
+
           if (!body && statusCode >= 200 && statusCode <= 206) {
             // adds status and data properties so JSON.parse doesn't throw an error
             body = `{
               "status": "success",
-              "data": "statusCode: ${statusCode}, statusMessage: ${statusMessage}"
+              "data": "statusCode: ${statusCode} -- statusMessage: ${statusMessage}"
             }`;
+          } else if (statusCode === 400) {
```

Comment:
@MarcusHurney are you sure the expected statusCode is `400`?
In the docs it says it should be `404` - AKA not-found?
![404](https://user-images.githubusercontent.com/376611/45082833-8b4a1200-b0fa-11e8-839c-1d7e41cb4ec1.png)

Also, I am a bit worried that any other status code 400/404 will not be properly handled with this if clause. E.g. other endpoints might have status 400 as real error?